api/service/stat.go

```diff
@@ -40,7 +40,8 @@ func (s *statService) Get(key string) (*PairStats, error) {
 
 	switch key {
 	case statPeriod24h:
-		err := s.sumPairStats24h(0, pairStatMap)
+		minTs := time.Now().Add(-24 * time.Hour)
```

Comment:
nit.
```suggestion
		minTs := time.Now().Sub(24 * time.Hour)
```

Reply:
`Sub` needs `time.Time`, not `time.Duration`